src/msg/async/Stack.cc

```diff
@@ -198,7 +198,7 @@ class C_drain : public EventCallback {
 
 void NetworkStack::drain()
 {
-  ldout(cct, 10) << __func__ << " started." << dendl;
+  ldout(cct, 21) << __func__ << " started." << dendl;
```

Comment:
@yuyuyu101 
I set these all to 21 to just get them out of the "regular" full debug mode.
But 30 is fine with me as well.
Will fix